vscode/src/copilot/webview/hljs-qsharp.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { HLJSApi, Mode, type Language } from "highlight.js";
+
+// NOTE: This has a bunch of differences from how we structure the language in <vscode/syntaxes/qsharp.tmLanguage.json>
+// We should align with that as much as possible eventually.
+
+const KEYWORDS_GENERAL = [
+  "use",
+  "borrow",
+  "mutable",
+  "let",
+  "set",
+  "if",
+  "elif",
+  "else",
+  "repeat",
+  "until",
+  "fixup",
+  "for",
+  "in",
+  "while",
+  "return",
+  "fail",
+  "within",
+  "apply",
+];
+
+const KEYWORDS_DECL = [
+  "namespace",
+  "open",
+  "import",
+  "export",
+  "as",
+  "internal",
+  "newtype",
+  "struct",
+  "operation",
+  "function",
+  "new",
+  "body",
+  "adjoint",
+  "Adjoint",
+  "controlled",
+  "Controlled",
+  "self",
+  "auto",
+  "distribute",
+  "invert",
+  "intrinsic",
+];
+
+const KEYWORDS_TYPE = [
+  "Int",
+  "BigInt",
+  "Double",
+  "Bool",
+  "Qubit",
+  "Pauli",
+  "Result",
+  "Range",
+  "String",
+  "Unit",
+  "Ctl",
+  "Adj",
+  "is",
```

Comment:
These categories look fairly loose, but I'd probably put `is` under "decl" rather than "type".

(I'm ok if you disagree and leave it - if we were to start nitpicking, we'd probably end up defining like twelve separate categories. Let's not.)

Reply:
Copied from our VS Code syntax <https://github.com/microsoft/qsharp/blob/main/vscode/syntaxes/qsharp.tmLanguage.json#L165>. I'd rather keep consistent for now.